modules/rostests/kmtests/tcpip/TcpIp_user.c

```diff
@@ -57,7 +57,7 @@ AcceptProc(
     ok(Error == 0, "");
 
     ListenSocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
-    ok_bool_true(ListenSocket != INVALID_SOCKET, "socket failed");
+    ok(ListenSocket != INVALID_SOCKET, "socket failed");
```

Comment:
This is now missing its \n

Reply:
.. which by the way is a perfect example of why you shouldn't make unnecessary changes. If a trivial two line change can introduce a regression, think about what useless refactoring can do.

Reply:
Ouch.

Reply:
@ThFabba I will revert it then.

Reply:
The change is fine, you could just add the \n instead.

Reply:
Eh, no `git-revert` practice then 😋

Reply:
:D
Actually there are tons more \n-less ok()s in this file. You could fix those too if you wanted.

Reply:
I guess I'll leave that to @SergeGautherie 😄

Reply:
> you shouldn't make unnecessary changes

I know. I try to, though I do not always agree. (I don't like stalled "wrong" code.)
I shall take part of the blame for this regression.

> there are tons more \n-less ok()s in this file

I would fix the whole file.
Nonetheless, in this very case, I would rather say the kmtest harness is inconsistent and error-prone, obviously.
Then, I would rather (soon) fix the latter: to add a `\n` in `KmtOk(), KmtTrace() and KmtSkip()`.

Reply:
> I would rather (soon) fix the latter

See #349.